### DISCUSSION TITLE
Will 'list_objects_version: "v1"' safely fix "Truncated Response should have continuation token set" SAFELY?

### DISCUSSION THREAD

**Discussion by jamesbot9000** [thanos-io/thanos#5249] [Questions & Answers]
We're running Thanos 0.17, we're seeing the compactor fail to delete partially uploaded blocks with 

"Truncated response should have continuation token set" 

From reading around it seems we need to set this on our S3 config:

list_objects_version: "v1"

We don't have that set now, and our object store (ceph) doesn't support listobjectsv2 on its current version.  The storage consumption seems to be growing faster than it should due to these failed compaction deletes. If we stop the compactor, add this config, and restart, will that be safe to do? We run our compactor as a singleton via systemd and we believe this will be safe, but this is our first time changing config since this was set up and want to make sure. We would really like to avoid damaging any production thanos data. 

Happy to provide any additional information. 

Thanks!

**Comment by GiedriusS** [comment]
I've been using Ceph with this option for years for this exact reason so it's safe to say that everything should be OK.
